api_v3/lib/KalturaLiveEntryService.php

```diff
@@ -545,17 +547,19 @@ protected function checkAndCreateRecordedEntry($dbLiveEntry, $mediaServerIndex,
 					$isNewSession = $dbLiveEntry->getLastBroadcastEndTime() + kConf::get('live_session_reconnect_timeout', 'local', 180) < $dbLiveEntry->getCurrentBroadcastStartTime();
 					$recordedEntryNotYetCreatedForCurrentSession = $recordedEntryCreationTime + kConf::get('live_session_reconnect_timeout', 'local', 180) < $dbLiveEntry->getCurrentBroadcastStartTime();
 					$maxAppendTimeReached = ($recordedEntryCreationTime + self::SEVEN_DAYS_IN_SECONDS) < time();
+					$recordingMoreThenLimit = $dbRecordedEntry->getDuration() > kConf::get('recording_limit_length', 'local', 86400);
```

Comment:
Use predefine const
For map name - use kConfMapNames (need to add the local)
'recording_limit_length' add as const on liveentry object